src/components/Calculator/Calculator.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Calculator.css';
+
+const Calculator = () => (
```

Comment:
Great job @hajnaloltyan 🤝

- [ ] As stated in the project requirements, kindly ensure the Calculator.js should have at least 1 child component.